client/containers/DashboardSettings/CollectionSettings/CollectionMetadataEditor.tsx

```diff
@@ -224,7 +225,9 @@ class CollectionMetadataEditor extends React.Component<Props, State> {
 		return (
 			<div className="dashboard-content_collection-metadata-editor">
 				{this.renderFields()}
-				<ButtonGroup>{this.renderGetDoiButton()}</ButtonGroup>
+				{this.props.allowUpdateDoi !== false && (
+					<ButtonGroup>{this.renderGetDoiButton()}</ButtonGroup>
+				)}
```

Comment:
A new prop called `allowUpdateDoi` was added to the collection metadata editor to disable DOI generation when the community is configured to deposit to Datacite, since it should be handled by the new Datacite deposit UI.